src/MultivariatePolynomials.jl

```diff
@@ -6,7 +6,20 @@ module MultivariatePolynomials
 
 import Base: *, +, -, /, ^, ==,
     promote_rule, convert, show, isless, size, getindex,
-    one, zero, transpose, isapprox, @pure, dot, copy
+    one, zero, isapprox, @pure, copy
+using Compat
+import Compat.LinearAlgebra: dot, norm
```

Comment:
I keep getting warnings
```
WARNING: Base.dot is deprecated: it has been moved to the standard library package `LinearAlgebra`.
Add `using LinearAlgebra` to your imports.
  likely near /home/blegat/.julia/v0.7/MultivariatePolynomials/test/division.jl:1
```
even with this. Do you get these too ?

Reply:
You might just need to upgrade your version of Compat.